browser/help/topics/clinvar-hts.md

```diff
@@ -0,0 +1,84 @@
+---
+id: clinvar-hts
+title: 'ClinVar Hail Tables'
+---
+
+### Overview
+
+We release two data tables underlying the ClinVar data displayed in the gnomAD browser. On a bi-monthly basis, we process the ClinVar monthly VCV XML release into two Hail Tables, one for GRCh38 and one for GRCh37. These tables enable our users to more easily incorporate ClinVar data into external pipelines in a manner consistent with what they see in the browser.
+
+These tables are stored in a requester pays GCS bucket. As such, to access or download this data you must provide a billing project when accessing the data with `gsutil`, e.g.
+
+```
+gsutil -u YOUR_PROJECT -m cp -r \
+  gs://gnomad-browser-clinvar/gnomad_clinvar_grch38.ht \
+  gs://YOUR-BUCKET/YOUR-OPTIONAL-NESTED-BUCKETS/gnomad_clinvar_grch38.ht
+```
+
+<br />
+
+#### ClinVar GRCh38 Hail Table annotations
+
+Global Fields:
+
+- `clinvar_release_date`: Release date of the ClinVar VCV XML file used to generate this Hail Table.
```

Comment:
`gs://gnomad-browser-clinvar/gnomad_clinvar_grch38.ht` currently also has this global: `mane_select_version`. will that be removed for future releases? if not, we should probably add it here